### DISCUSSION TITLE
Win10: Crash on pen input (pressure stylus pen)

### DISCUSSION THREAD

**Discussion by 2Kbummer** [xournalpp/xournalpp#3117] [General]
I bought a drawing tablet (GAOMON S620 Graphics Tablet @ amazon). Found this. Downloaded windows release. Used the mouse to draw, works fine. Then used pressure stylus pen and crashes program upon input (closes and doesnt show save prompt). I messed with the settings, and nothing works. Tried .20 and .19 version. Both crash. Recognizes pen, and can move mouse cursor, but as soon as it touches tablet, program crashes. Only happens with this program.

**Comment by rolandlo** [comment]
Have you also tried version [1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly)?

**Reply by 2Kbummer** [reply]
Tried it. Same problem.

**Comment by rolandlo** [comment]
Do you have some errorlogs (in the Appdata\Local\Xournal++ folder, I think)?

**Reply by 2Kbummer** [reply]
the errorlogs folder is empty

**Comment by rolandlo** [comment]
Seems you are not the only one with this problem on a GAOMON tablet, see #2719 and #1788 and #2739. It looks like a driver problem. Do you have any other software based on Gtk/Gdk for your tablet?

**Reply by 2Kbummer** [reply]
Thats Unfortunate. Im not sure. I only used it on OneNote. Im going to try "krita" drawing program.

**Reply by 2Kbummer** [reply]
pen works for open-source "krita" drawing program. not sure if its based off Gtk/Gdk.

**Reply by rolandlo** [reply]
Krita is based on Qt, not on Gtk. Maybe try Inkscape and/or GIMP.

**Reply by 2Kbummer** [reply]
works on gimp. not sure how to turn on the pressure input on gimp though.

**Comment by rolandlo** [comment]
Ok, GIMP still uses GTK+ 2, while Xournal++ and Inkscape use GTK+ 3. Maybe you find some useful info on the Inkscape forum. Here is a starter: https://inkscape.org/ar/forums/questions/inkscape-crashing-as-soon-as-i-use-my-tablet/

**Reply by 2Kbummer** [reply]
yup crashes on inkscape. When i hover over the tablet an imediate line appears, one already anchored on the page , and the other end attached to the hovering pen cursor. upon touching the tablet, it crashes.

**Reply by rolandlo** [reply]
Another program also using GTK3 is MyPaint. See https://github.com/mypaint/mypaint/issues/1126

**Reply by 2Kbummer** [reply]
i closed the tablet software and now can draw on xournal, but with input issues. Sometimes registers 1 drawn line as just 1 dot. So drawing the word "the" , some strokes are missing and only registers the intitial touch, making a dot instead of a line.

**Reply by rolandlo** [reply]
Does that happen even if you turn off pressure sensitivity (in the Stylus panel of menu `Edit > Preferences`)?

**Reply by 2Kbummer** [reply]
yup. disabled. closed application. restarted application. tested. same crash.

**Comment by rolandlo** [comment]
Here is a link to the upstream issue: https://gitlab.gnome.org/GNOME/gtk/-/issues/3860

**Reply by 2Kbummer** [reply]
look like it is still unsolved :(

**Reply by rolandlo** [reply]
@2Kbummer Can you follow [this advice](https://github.com/xournalpp/xournalpp/issues/2316#issuecomment-873435907) and check if the patched Gtk solves the issue?

**Comment by 2Kbummer** [comment]
Returned product , don't have anymore

________________________________

**Comment by rolandlo** [comment]
This is probably fixed in [version 1.1.0-hotfix-1](https://github.com/xournalpp/xournalpp/releases/tag/1.1.0-hotfix.1)

**Reply by cklongg** [reply]
Does Xournal++ still support win 7 (64bit) users? I have same crash problem so the PC with win 10's problem was fixed by version 1.1.0-hotfix-1 but my laptop with win 7 still have problem. I use Gaomon 1060 Pro and Gaomon driver v16.0.0.30.

**Comment by nianyu87** [comment]
I also have a [gaomon tablet](https://pctechtest.com/best-gaomon-drawing-tablets), it works well with Xournal++ for note taking in windows 11.
